### PR TITLE
add getLatestRelease.bat, .gitignore

### PR DESCRIPTION
**Pull request by Rhyknowscerious**
My first pull request. Let me know if I'm missing anything.
Added a bat script. Call it like this:

CALL getLatestRelease.bat https://github.com/yt-dlp/yt-dlp/releases/latest yt-dlp_win.zip

### CODE REVIEW

**Comment by Rhyknowscerious** [comment]
This should work with other "latest release" links and download target from that page.

**Comment by Rhyknowscerious** [comment]
Got it up to date and tested. Should work 100% now. It does not fit within the current dimension constraints in the terminal, however.

File: src/getLatestRelease.bat [hunk 1]
```diff
@@ -4,47 +4,97 @@ REM Notes
...
+    SET "downloadDestinationFolder="
+    SET "downloadDestinationFullName="
+    SET "targetFileName="
     SET "latestReleaseRedirectURL="
     SET "latestReleaseURL="
 
-    REM Process argument defaults for numbered parameters 1 and 2
+    REM Process argument defaults for numbered parameters
     IF [%1] == [] (
+        <NUL SET /P "=getLatestRelease.bat: setting default value for latestReleaseURL: "
```
Comment (Rhyknowscerious) [inline src/getLatestRelease.bat:23]: Print to console without a new line

File: src/yt-dlp CLI.bat [hunk 2]
```diff
@@ -16,9 +16,11 @@ goto :yt-dlpCheck
 
 :Downloadyt-dlp
 cls & mode con cols=58 lines=4 & title yt-dlp CLI & color 0e & echo.
-echo 1. Download yt-dlp_win.zip
-echo 2. Extract contents to .\bin\ and restart script.
-start https://github.com/yt-dlp/yt-dlp/releases/latest/
+REM Fully automate the download and extraction with Windows built-in tools (Specific release of Windows 10+ Required)
+REM https://learn.microsoft.com/en-us/virtualization/community/team-blog/2017/20171219-tar-and-curl-come-to-windows
+CALL getLatestRelease.bat "https://github.com/yt-dlp/yt-dlp/releases/latest" "yt-dlp_win.zip" "%userprofile%\Downloads" || (
+    <NUL SET /P "=Download failed. " & PAUSE & EXIT
```
Comment (Rhyknowscerious) [inline src/yt-dlp CLI.bat:22]: Is it ok to call the other script externally like this or would you rather have all the code in one script file? And if you prefer it all in a single file, should I refactor getLatestRelease.bat to match the coding style of yt-dlp CLI.bat file?

Comment (MrMendelli) [reply src/yt-dlp CLI.bat:22]: I would prefer for the update function to be a builtin feature to the main script as an additional main menu option.

**Pull request closed without merging**